src/main/java/StrategyPatternClasses/TableGenerator.java

```diff
@@ -0,0 +1,94 @@
+package StrategyPatternClasses;
+
+import automatabuilder.Symbol;
+import interfaces.IAlphabet;
+import interfaces.IState;
+import interfaces.ITransition;
+
+import java.util.List;
+
+/**
+ * Created by oliv on 5/23/15.
+ */
+class TableGenerator {
+    StringBuilder sb            = new StringBuilder();
+    StringBuilder whitespace    = new StringBuilder();
+    String whitespaceString;
+
+    public String getTable(List<IState> stateList, IAlphabet alphabet){
```

Comment:
Instead of returning a string, maybe supplying a PrintStream to the TableGenerator?
```Java
class TableGenerator {
   private PrintStream out;
   ...
   
    public TableGenerator(PrintStream out) {
        if (out == null) throw new Exception();
        this.out = out;
    }
    public void generate() {
        String table = getTable(...);
        out.write(table);
    }
    ...

}
```